content/src/migrations/scripts/1646919812526_create_active_pointers.ts

```diff
@@ -0,0 +1,252 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('active_pointers', {
+    pointer: { type: 'text', primaryKey: true },
```

Comment:
Please consider using `varchar` instead of `text` for primary key performance and storage.

Also consider adding a GIN trgm or text_pattern index for LIKE operations:

https://www.postgresql.org/docs/9.1/pgtrgm.html

```sql
CREATE INDEX active_pointers_pointers_ops  ON active_pointers USING gin  (pointer gin_trgm_ops);
-- OR
CREATE INDEX active_pointers_pointers_pattern ON active_pointers(pointer text_pattern_ops);
```

Reply:
The search is case-sensitive, is that a problem? I'm not sure if I want to add the `LOWER` directive in the query 🤔